util/shlib_wrap.sh.in

```diff
@@ -117,6 +117,9 @@ if [ -f "$LIBCRYPTOSO" -a -z "$preload_var" ]; then
 fi
 {- output_on() unless grep (/-rpath\b/, @{$config{LDFLAGS}}); ""; -}
 
+OPENSSL_MODULES="${THERE}/providers"
+export OPENSSL_MODULES
```

Comment:
Yeah I know, it's practical...  but considering `OPENSSL_ENGINES` isn't defined here, I see no reason why `OPENSSL_MODULES` should...

Reply:
I'd say the fact that OPENSSL_ENGINES isn't is wrong.

Reply:
That's a point of view.  This script is a *shlib* wrapper, which is arguably not relevant for dynamically loadable libraries.